cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -0,0 +1,1006 @@
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'organism_ontology_term_id' is not a valid "
+                "ontology term id of 'NCBITaxon'"
+            ],
+        )
+
+    def test_tissue_ontology_term_id(self):
+
+        """
+        tissue_ontology_term_id categorical with str categories. This MUST be the term that best describes the tissue
+        that this cell was derived from, depending on the type of biological sample:
+            Cell Culture - MUST be a CL term appended with " (cell culture)"
+            Organoid - MUST be an UBERON term appended with " (organoid)"
+            Enriched, Sorted,or Isolated Cells from a Tissue - MUST be an UBERON or CL term and SHOULD NOT use terms
+                that do not capture the tissue of origin (e.g. In the case of CD3+ kidney cells, use "UBERON:0002113"
+                for kidney instead of "CL:000084" for T cell. However, in the case of EPCAM+ cervical cells,
+                use "CL:000066" for epithelial cell of the cervix.)
+        """
+
+        self.validator.adata.obs["tissue_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'tissue_ontology_term_id' is not a "
+                "valid ontology term id of 'UBERON, CL'"
+            ],
+        )
+
+        # Cell Culture - MUST be a CL term appended with " (cell culture)"
+        self.errors = []
+        self.validator.adata.obs["tissue_ontology_term_id"][
+            0
+        ] = "CL:0000057 (CELL culture)"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'CL:0000057 (CELL culture)' in 'tissue_ontology_term_id' is "
+                "not a valid ontology term id of 'UBERON, CL'"
+            ],
+        )
+
+        # Organoid - MUST be an UBERON term appended with " (organoid)"
+        self.errors = []
+        self.validator.adata.obs["tissue_ontology_term_id"][0] = "CL:0000057 (ORGANOID)"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'CL:0000057 (ORGANOID)' in 'tissue_ontology_term_id' is "
+                "not a valid ontology term id of 'UBERON, CL'"
+            ],
+        )
+
+    def test_sex_ontology_term_id(self):
+
+        """
+        sex_ontology_term_id categorical with str categories.
+        This MUST be a child of PATO:0001894 for phenotypic sex or "unknown" if unavailable
+        """
+
+        self.validator.adata.obs["sex_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'sex_ontology_term_id' is "
+                "not a valid ontology term id of 'PATO'"
```

Comment:
```suggestion
                "neither a valid ontology term id of 'PATO' nor the value 'unknown'"
```

Reply:
Fixed in f68682ff4c918f36b5fae8f4fab1d2b98201caf4